go.mod

```diff
@@ -4,13 +4,13 @@ go 1.18
 
```

Comment:
you might want to bump to `go 1.19` as that's circl compatible version.

Reply:
done